pkg/opte.template.p5m

```diff
@@ -32,5 +32,6 @@ file path=kernel/drv/amd64/xde owner=root group=sys mode=0755 \
     pkg.depend.bypass-generate=.*dld.* \
     pkg.depend.bypass-generate=.*mac.*
 driver name=xde
+link path=dev/xde target=../devices/pseudo/xde@0:ctl
```

Comment:
The @dld@ one is a bit suspicious and unfortunate. We probably should file an illumos issue to have that handled by devfsadm or others. The only reason I can imagine it being there otherwise is some part of early boot time handling ala the bits that @jclulow did elsewhere.

In terms of what you can do, what I've done in the past is delivered my own devfsadm plugin that has this definition. devfsadm is built up as a series of plugins that it loads and it can be delivered by the driver package (while that's kind of a private interface, we're already using private interfaces all over this). If you'd like additional pointers / examples, let me know. We can file a follow up to go deal with this to help unblock things if that's helpful.

Reply:
Opened upstream bug for dld: https://www.illumos.org/issues/15337

Reply:
Added devfsadm plugin to handle the `/dev/xde` link creation/removal.